ios/Classes/SwiftHmssdkFlutterPlugin.swift

```diff
@@ -1140,12 +1161,12 @@ public class SwiftHmssdkFlutterPlugin: NSObject, FlutterPlugin, HMSUpdateListene
                     if self.audioMixerSourceMap[node] == nil {
                         if node=="mic_node" {
                             self.audioMixerSourceMap["mic_node"] = HMSMicNode()
-                        } else if node == "screen_broadcast_audio_receiver_node" {
-                            // TODO: add checks for 100ms App Group for Audio Share
+                        } else if node == "screen_broadcast_audio_receiver_node" && sdk.appGroup != nil {
```

Comment:
Verify the behaviour of `sdk.appGroup != nil` when appGroup is passed/skipped in both older appGroup & the new IOSScreenshareConfig